scripts/post-install/post-install.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const Logger = require('../utils/logger');
+
+fs.readFile(`${process.env.INIT_CWD}/package.json`, (error, file) => {
+  if (error) {
+    throw Logger.error(error);
+  }
+
+  const { dependencies } = JSON.parse(file);
+
+  if (dependencies['terra-toolkit']) {
+    Logger.warn('+-------------------------------------------------------------------------------------------------------------+');
+    Logger.warn('|                                                   WARNING                                                   |');
+    Logger.warn('+-------------------------------------------------------------------------------------------------------------+');
+    Logger.warn('|                            terra-toolkit should be installed as a devDependency.                            |');
+    Logger.warn('|                                                                                                             |');
+    Logger.warn('| Modify the package.json to remove terra-toolkit from dependencies and add terra-toolkit to devDependencies. |');
+    Logger.warn('+-------------------------------------------------------------------------------------------------------------+');
```

Comment:
Formatting and warning message suggestions are welcome. The table can format strangely on small viewports, but it does help it stand out a little more in the terminal output.

Reply:
Should we state that this will fail install in the future? Even if it's just a threat? 😈